src/main/java/novamachina/exnihilosequentia/common/item/resources/EnumResource.java

```diff
@@ -8,7 +8,7 @@
 public enum EnumResource implements IExtensibleEnum {
     ANCIENT_SPORE(Items.ANCIENT_SPORE),
     GRASS_SEED(Items.GRASS_SEED),
-    SILKWORM(Items.SILKWORM),
+    //SILKWORM(Items.SILKWORM),
```

Comment:
Why are we removing the silkworm from here?

Reply:
just because of creative tab order, i added it directly to ExNihiloItems

Reply:
I've removed silkwork out of enumresources